lib/syntax_tree/node.rb

```diff
@@ -6264,7 +6175,7 @@ def format(q)
         # If we can transform this node into a ternary, then we're going to
         # print a special version that uses the ternary operator if it fits on
         # one line.
-        if Ternaryable.call(q, node)
+        if Ternaryable.call(q, node) && keyword != "elsif"
```

Comment:
One of the test cases was turning an `elsif` into a ternary, so I added this, but happy to modify the `Ternatyable.call` method in some way to prevent this from happening instead if that makes more sense.

Reply:
I think it shouldn't even get into here honestly. I think you can just copy-paste what used to be in the elsif formatting and put it into the `If` node's formatting function, and then call into `ConditionalFormatter` if it's not an elsif. That makes it so you don't have to have all of these checks floating around.